storage/mysql/storage.sql

```diff
@@ -55,10 +55,11 @@ CREATE TABLE IF NOT EXISTS TreeHead(
   RootSignature        VARBINARY(1024) NOT NULL,
   TreeRevision         BIGINT,
   PRIMARY KEY(TreeId, TreeHeadTimestamp),
-  UNIQUE INDEX TreeRevisionIdx(TreeId, TreeRevision),
   FOREIGN KEY(TreeId) REFERENCES Trees(TreeId) ON DELETE CASCADE
 );
 
+CREATE UNIQUE INDEX TreeHeadRevisionIdx
```

Comment:
(curiosity) is there a material difference between this and how it was declared before?

Reply:
Apart from having an explicit name, I think it's the same.